.github/workflows/rust.yml

```diff
@@ -41,12 +43,21 @@ jobs:
 
       - name: Cache Cargo registry
         uses: Swatinem/rust-cache@v2
+        with:
+          shared-key: 'rust-ci'
+          cache-bin: 'false'
+
+      - name: Run sccache-cache
+        uses: mozilla-actions/sccache-action@v0.0.9
 
       - name: Install cargo-nextest
         uses: taiki-e/install-action@v2
         with:
           tool: nextest
 
+      - name: Fetch dependencies
+        run: cargo +${{ matrix.rust }} fetch --locked
```

Comment:
What does this change?

Reply:
It's not required; I extracted it to see how long the build takes on its own. It doesn't hurt to leave it though